src/main/java/LottoApplication.java

```diff
@@ -0,0 +1,54 @@
+import java.util.ArrayList;
+import java.util.List;
+import domain.LottoNumber;
+import domain.Lotto;
+import domain.LottoCount;
+import domain.Lottos;
+import domain.Money;
+import domain.ProfitResult;
+import domain.WinningLotto;
+import ui.Receiver;
+import ui.Printer;
+
+public class LottoApplication {
+
+    Printer printer = new Printer();
+    Receiver receiver = new Receiver();
```

Comment:
```suggestion
    private final Printer printer = new Printer();
    private final Receiver receiver = new Receiver();
```

main이니 큰 문제는 없겠지만, 그래도 오용을 막기위해선 private final을 붙여주는게 안전할것 같네요 🙂